linkage_dag.py

```diff
@@ -341,19 +342,55 @@
         ),
     ]
 
-    # Start up godzilla of article linkage, create the merged ids
+    gce_instance_create = ComputeEngineInsertInstanceOperator(
+        task_id=f"create_{gce_resource_id}",
+        project_id=PROJECT_ID,
+        zone=GCP_ZONE,
+        body={
+            "name": gce_resource_id,
+            "machine_type": f"zones/{GCP_ZONE}/machineTypes/m1-megamem-96",
+            "disks": [
+                {
+                    "boot": True,
+                    "auto_delete": True,
+                    "initialize_params": {
+                        "disk_size_gb": "500",
+                        "disk_type": f"zones/{GCP_ZONE}/diskTypes/pd-balanced",
+                        "source_image": "projects/ubuntu-os-cloud/global/images/ubuntu-2204-jammy-v20240927",
```

Comment:
Are we going to have to regularly update this source image, since it's from September 2024? Or are we assuming that doesn't really matter since we kept a machine from much longer ago with no issue? (will these images even last to keep being created?)

Reply:
Good question - my understanding (and observation from listing the images) is these images stick around long-term. We may want to occasionally update this regardless, though